dap/src/responses.rs

```diff
@@ -46,12 +46,14 @@ pub struct CompletionsResponse {
 }
 
 #[derive(Serialize, Debug, Default, Clone)]
+#[serde(rename_all = "camelCase")]
 #[cfg_attr(feature = "client", derive(Deserialize))]
 #[cfg_attr(feature = "integration_testing", derive(Dummy))]
 pub struct ContinueResponse {
   /// The value true (or a missing property) signals to the client that all
   /// threads have been resumed. The value false indicates that not all threads
   /// were resumed.
+  #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Adding `camelCase` here resulted in a test failure of:

```
test integration_tests::validate_continue_response ... FAILED

failures:

---- integration_tests::validate_continue_response stdout ----
Validation error: null is not of type "boolean"
Instance path: /body/allThreadsContinued
thread 'integration_tests::validate_continue_response' panicked at /Users/davis/files/programming/dap-rs/target/debug/build/integration_tests-c7326a24e20e3942/out/generated_tests.rs:229:5:
assertion failed: compiled.is_valid(&instance)
```

which I eventually figured out meant that `allThreadsContinued` should simply not be serialized as anything if it is `None`. It seems like there are some other places in the library where you do this already.